From d5c7a43eba53a421908bfcc5d84419fbab1b2528 Mon Sep 17 00:00:00 2001
From: AlexYaruki <alexyaruki@gmail.com>
Date: Fri, 2 Sep 2016 12:45:46 +0200
Subject: [PATCH] Fix to inlined functions definition

Fixes:

    Error relocating bindings/java/sigar-bin/lib/libsigar-amd64-linux.so: sigar_skip_multiple_token: symbol not found
    Error relocating bindings/java/sigar-bin/lib/libsigar-amd64-linux.so: sigar_skip_token: symbol not found

diff --git a/include/sigar_util.h b/include/sigar_util.h
index b3c4dd7d..9545d0f6 100644
--- a/include/sigar_util.h
+++ b/include/sigar_util.h
@@ -75,11 +75,11 @@ int sigar_inet_ntoa(sigar_t *sigar,
 struct hostent *sigar_gethostbyname(const char *name,
                                     sigar_hostent_t *data);
 
-SIGAR_INLINE char *sigar_skip_line(char *buffer, int buflen);
+extern SIGAR_INLINE char *sigar_skip_line(char *buffer, int buflen);
 
-SIGAR_INLINE char *sigar_skip_token(char *p);
+extern SIGAR_INLINE char *sigar_skip_token(char *p);
 
-SIGAR_INLINE char *sigar_skip_multiple_token(char *p, int count);
+extern SIGAR_INLINE char *sigar_skip_multiple_token(char *p, int count);
 
 char *sigar_getword(char **line, char stop);
 
